packages/libsql-client/src/sqlite3.ts

```diff
@@ -109,17 +121,20 @@ export class Sqlite3Client implements Client {
     async execute(stmt: InStatement): Promise<ResultSet>;
     async execute(sql: string, args?: InArgs): Promise<ResultSet>;
 
-    async execute(stmtOrSql: InStatement | string, args?: InArgs): Promise<ResultSet> {
-      let stmt: InStatement;
+    async execute(
+        stmtOrSql: InStatement | string,
+        args?: InArgs,
+    ): Promise<ResultSet> {
+        let stmt: InStatement;
 
-      if (typeof stmtOrSql === 'string') {
-        stmt = {
-            sql: stmtOrSql,
-            args: args || []
-        };
-      } else {
-          stmt = stmtOrSql;
-      }
+        if (typeof stmtOrSql === "string") {
+            stmt = {
+                sql: stmtOrSql,
+                args: args || [],
+            };
+        } else {
+            stmt = stmtOrSql;
+        }
```

Comment:
This looks like pure whitespace change? Please don't mix fixes and cleanups in one pull request. It just makes reviewing harder.